packages/ceramic-cli/src/index.ts

```diff
@@ -18,9 +19,13 @@ program
   .action(async (doctype, content, { onlyGenesis }) => {
     content = JSON.parse(content)
     const ceramic = new CeramicClient()
-    const doc = await ceramic.createDocument(content, doctype, { onlyGenesis })
-    console.log(doc.id)
-    console.log(JSON.stringify(doc.content, null, 2))
+    try {
+      const doc = await ceramic.createDocument(content, doctype, { onlyGenesis })
+      console.log(doc.id)
```

Comment:
We can introduce a specific logger (e.g. `winston`) instead of directly using `console.log`

Reply:
Definitely, created an issue here: https://github.com/ceramicnetwork/js-ceramic/issues/8